config/locales/en/course/assessment/assessments.yml

```diff
@@ -60,6 +60,10 @@ en:
             password (with the help of course staff). In addition, only one session is allowed to
             access the submission at any given time.
           password_placeholder: 'Leave blank to disable password protection'
+          delayed_grade_publication: 'Enable Delayed Grade Publication'
+          delayed_grade_publication_hint: >
+            With this enabled, the grades of all the submissions won't be immediately published to
+            students, you can publish them all together at a later time at the submissions page.
```

Comment:
With this enabled, gradings done by course staff will not automatically be published. To publish all the gradings for this assessment, you can do so at a later time at the submissions page.